src/index.scss

```diff
@@ -12,6 +12,10 @@ body {
   font-family: 'Raleway', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+  user-select: none;
```

Comment:
Hhhm... you really want to set this globally? On which element do you want to apply this rule? I suggest to further specify the use

Reply:
The specific reason why I've set this property was to prevent random selections of areas/text on mobile phones while long-pressing/holding a button to see it's tooltip.

Reply:
Ok, got it. Then you should set this on the `.menu-item` class I guess so it won't affect any other area

Reply:
I removed it because the random selection hasn't happened anymore on my phone